listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -14,6 +14,7 @@ import FollowUsers from "./FollowUsers";
 import APIService from "./APIService";
 import Loader from "./components/Loader";
 import ListenCard from "./listens/ListenCard";
+import FollowerFollowingEntity from "./follow/followerFollowingEntity";
```

Comment:
I added a basic UserFeed page in this commit (https://github.com/metabrainz/listenbrainz-server/pull/1109/commits/7d1f5f5e5637ddf021dea063f75bfe3b838759cf), you could cherry pick the commit and then show this component there instead of adding it to the recent listens page.